tests/test_ui.py

```
@@ -355,45 +311,46 @@ def test_steps_render_1_failure_paths():
-            return ["Score1"]
-        return default
+    mock_state = DummySessionState(
+        {
+            "participants_df": mock_df,
+            "score_cols": ["Score1"],
+            "warm_start_cache": {},
+        }
+    )
 
-    mock_state.get.side_effect = get_side_effect
+    def mock_cols(n, **_):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `mock_cols`

(ANN202)